src/main/java/DukeException.java

```diff
@@ -0,0 +1,37 @@
+public class DukeException extends Throwable {
+
+    String s;
```

Comment:
Perhaps you can name your variable to a noun instead of using a scratch variable as stated in the style guide.
```suggestion
    String userInput;
```